scripts/setup_native_deps.sh

```diff
@@ -16,17 +16,17 @@ setup_llvm_deps() {
 	case "$(uname)" in
 	Darwin)
 		brew update
-		brew install llvm@18
+		brew install llvm@19
 
 		LIBRARY_PATH=/opt/homebrew/lib
-		MLIR_SYS_180_PREFIX="$(brew --prefix llvm@18)"
-		LLVM_SYS_181_PREFIX="$MLIR_SYS_180_PREFIX"
-		TABLEGEN_180_PREFIX="$MLIR_SYS_180_PREFIX"
+		MLIR_SYS_190_PREFIX="$(brew --prefix llvm@18)"
```

Comment:
This should be `@19`. 
```suggestion
		MLIR_SYS_190_PREFIX="$(brew --prefix llvm@19)"
```